src/functions/containers/FunctionEditPage.tsx

```diff
@@ -0,0 +1,25 @@
+// Libraries
+import React, {PureComponent} from 'react'
+
+// Components
+import {Page} from '@influxdata/clockface'
+
+// Utils
+import {pageTitleSuffixer} from 'src/shared/utils/pageTitles'
+
+class FunctionEditPage extends PureComponent {
```

Comment:
why not functional components for these?

Reply:
>why not functional components for these?

why functional components?

Reply:
we've been using them more and more for new components as an informal convention because hooks are dope and make state management tasks easier to implement without as much routing, curious what a pure component buys us / if it was intentional for some reason

Reply:
just being old-skool I guess! functionalized!